python/lsst/ip/isr/isrTask.py

```diff
@@ -904,9 +904,9 @@ def readIsrData(self, dataRef, rawExposure):
             - ``bfKernel``: Brighter-Fatter kernel (`numpy.ndarray`)
             - ``defects``: list of defects (`lsst.meas.algorithms.Defects`)
             - ``fringes``: `lsst.pipe.base.Struct` with components:
-                - ``fringes``: fringe calibration frame (`afw.image.Exposure`)
-                - ``seed``: random seed derived from the ccdExposureId for random
-                    number generator (`uint32`)
+              - ``fringes``: fringe calibration frame (`afw.image.Exposure`)
```

Comment:
``fringes``: `afw.image.Exposure`
Fringe calibration frame.
... would be more consistent, I think.

Reply:
This seems to be the way we document structs and dict-like objects.